.travis/prepare.sh

```diff
@@ -11,5 +11,7 @@ sudo tar -C /usr/local -xJf $CLANG_FILE
 sudo ln -s /usr/local/$CLANG_DIR /usr/local/clang
 rm $CLANG_FILE
 
+go get -u github.com/jteeuwen/go-bindata/...
```

Comment:
Can you document that somewhere? We already need `go-swagger` and now this `go-bindata` as well